apps/rush-lib/src/logic/ProjectWatcher.ts

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { FSWatcher } from 'chokidar';
```

Comment:
Use `Import.lazy` to load this.

Reply:
Using `await import()` to load ProjectWatcher from `BulkScriptAction`. Can also do this for a little extra.